src/lib/github-service.ts

```
@@ -128,9 +129,9 @@
       
       // Handle various GitHub URL formats with domain validation - using safer patterns
       const patterns = [
-        /^(?:https?:\/\/)?(?:www\.)?github\.com\/([^/]+)\/([^/]+)(?:\/.*)?$/,
-        /^(?:https?:\/\/)?(?:www\.)?github\.com\/([^/]+)\/([^/]+)\.git$/,
-        /^git@github\.com:([^/]+)\/([^/]+)\.git$/,
+        /^(?:https?:\/\/)?(?:www\.)?github\.com\/([a-zA-Z0-9_-]+)\/([a-zA-Z0-9_-]+)(?:\/.*)?$/,
```

ESLint

[error] Unsafe Regular Expression

(security/detect-unsafe-regex)